### PR TITLE
Use correct base path in webpack

### LINKED ISSUE #1179: Simorgh is using the incorrect URL for static assets
**Describe the bug**
Simorgh is currently requesting static assets from https://test.bbc.com, whoops

**To Reproduce**

1. Go to https://www.test.bbc.com/news/articles/c5v767l2l0zo
2. See error

image

**Expected behavior**
It should use `https://news.test.files.bbci.co.uk/include/articles/public/` which it gets from https://github.com/bbc/simorgh-infrastructure/blob/latest/Makefile#L7

- [x] Initially labelled with ["bug"](https://github.com/BBC-News/simorgh/labels/bug)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1179

Updates webpack to use correct base path as provided by simorgh-infra

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: webpack.config.client.js
Comment (pjlee11) [inline webpack.config.client.js]: As spoken about in person would be good to rename this to be more explicit such as `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` and then make it dependant on https://github.com/bbc/simorgh-infrastructure/pull/192/files

**Comment by dr3** [comment]
~Blocked On~ To be merged with https://github.com/bbc/simorgh-infrastructure/pull/198

**Review by pjlee11 — APPROVED** [review]
Nice one!

**Review by bcmn — APPROVED** [review]

**Comment by dr3** [comment]
Testing notes - This will primarily be tested once merged. Locally however static assets like the JS files should continue to be served from http://localhost:7080 ( when running build and start )

You can further check this by changing https://github.com/bbc/simorgh/pull/1180/files#diff-f579cccc964135c7d644c7b2d3b0d3ecR4 to  be `SIMORGH_PUBLIC_STATIC_ASSETS_PATH=http://127.0.0.1:7080` when built and run, the static assets should be requested from `http://127.0.0.1:7080` instead of localhost. This shows the correct value is being used in building.

**Comment by jamesbrumpton** [comment]
LGTM 👍

**Pull request merged**
